### DISCUSSION TITLE
关于适配了playerbot的az端，lua方面的问题，请帮忙看看，十分感谢。

### DISCUSSION THREAD

**Discussion by 13656694410** [mod-playerbots/mod-playerbots#190] [Q&A]
**Describe the bug**
您好，请问下，您适配了playerbot的az端，是否会出现下面这个问题：
我编译的时候加了eluna模块，在使用一些lua功能购买物品的时候会购买两次，扣费也会扣两次。比如签到lua、抽奖lua、商城lua之类的功能。我在UIWOW论坛上看到也有人提到过是适配了playerbot的az端的问题，换城az官方的就没问题了，我不知道他们说的端是不是您这个端，如有冒犯，还请原谅。

**Screenshots**
下面这张图是一个抽奖的lua，正常应该消耗1个公正徽章抽取一次物品，这里是消耗了两次。
![QQ图片20240306192826](https://github.com/liyunfan1223/mod-playerbots/assets/21214217/c91b4eca-8532-4e33-a8a2-07d2a3b006ce)

第二张图是一个签到的lua，正常是签到一次获得一次物品，这里是获得了两次。
![QQ截图20240306192615](https://github.com/liyunfan1223/mod-playerbots/assets/21214217/fd052ed3-9a1a-4764-8a42-040b79e94da3)

第三张图是一个商城的lua，也是购买一次变成消耗了两次并且获得了两次物品。
![QQ截图20240306192537](https://github.com/liyunfan1223/mod-playerbots/assets/21214217/52b23ff1-5f7b-403c-9ed8-e62f93ad7cfc)

请原谅我是个新手，github不太会使用，这个问题我想应该是在服务端那边提问，但是那边我没找到提问的地方，只能提问在此，抱歉抱歉。

**Comment by liyunfan1223** [comment]
Is it related to https://github.com/liyunfan1223/mod-playerbots/discussions/156? I'm not familiar with Eluna.
